src/components/PopupMenu.scss

```diff
@@ -0,0 +1,52 @@
+.popup {
+    position: relative;
+    user-select: none;
+}
+
+.popup__window {
+    position: absolute;
+    display: flex;
+    width: 180px;
+    height: 90px;
+    left: -160px;
+    top: -122px;
+    background: #FFFFFF;
+    box-shadow: 2px 0px 15px rgba(16, 19, 45, 0.1);
+    border-radius: 4px;
+    overflow: hidden;
+    a:hover {
+        background: #EEF3FE;
+    }
+}
+
+.popup__content {
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+    a {
+        display: flex;
+        flex-direction: column;
+        justify-content: center;
+        align-items: center;
+        padding: 0px 0px;
+        width: 180px;
+        height: 30px;
+        line-height: 30px;
+        text-indent: 16px;
+        font-family: 'ApercuPro';
```

Comment:
You can import the file "constants.scss" and write instead of font-family: "ApercuPro" - font-family: $font; but if you don't want to, please change the brackets to double, like:  ""